### DISCUSSION TITLE
Make use of All Contributors

### DISCUSSION THREAD

**Discussion by MagicLike** [JulianPrieber/littlelink-custom#147] [Feature requests]
Just an idea: What about making use of [All Contributors](https://allcontributors.org), to list the contributors in some way "automatically" instead of using a picture, which has to be replaced every time.
Also the type of contribution would be shown under each contributor.

**Comment by JulianPrieber** [comment]
Funny that you mention this. 
I use the picture **because** it is automated. I wrote a little PHP script that gets all GitHub contributors with an API request, and displays them as the image.

It appears the GitHub caching messes some stuff up. Changes in contributors take a long time to apply.

You can find the live version here: https://components.littlelink-custom.com/contributors/

I'm not sure why the version on GitHub lags so far behind, badges from https://shields.io/ appear to work just fine. This confuses me a bit.

**Reply by JulianPrieber** [reply]
I've read online that GitHub honors Cache-Control headers.
I added them to the PHP page, so this might work properly now.

**Reply by MagicLike** [reply]
Ahh, ok. I was just wondering why the picture lags behind the contributors page....
